apps/browser/src/admin-console/services/browser-policy.service.ts

```diff
@@ -17,11 +18,27 @@ export class BrowserPolicyService extends PolicyService {
   })
   protected _policies: BehaviorSubject<Policy[]>;
 
-  constructor(stateService: StateService, organizationService: OrganizationService) {
-    super(stateService, organizationService);
+  constructor(
+    stateService: StateService,
+    organizationService: OrganizationService,
+    autofillSettingsService: AutofillSettingsServiceAbstraction,
+  ) {
+    super(stateService, organizationService, autofillSettingsService);
     this._policies.pipe(this.handleActivateAutofillPolicy.bind(this)).subscribe();
   }
 
+  private async getActivateAutoFillOnPageLoadFromPolicy(): Promise<boolean> {
+    return await firstValueFrom(this.autofillSettingsService.activateAutoFillOnPageLoadFromPolicy$);
+  }
+
+  private async setActivateAutoFillOnPageLoadFromPolicy(autofillEnabled: boolean): Promise<void> {
+    this.autofillSettingsService.setActivateAutoFillOnPageLoadFromPolicy(!autofillEnabled);
+  }
```

Comment:
```suggestion
  private async setActivateAutoFillOnPageLoadFromPolicy(settingEnabled: boolean): Promise<void> {
    this.autofillSettingsService.setActivateAutoFillOnPageLoadFromPolicy(!settingEnabled);
  }
```

Nit: naming, the variable name here is kind of ambiguous.

Beyond that though, it seems kind of strange that we are setting the value as a negation `!autofillEnabled`.... I might not have context for why that would be necessary though, but it feels like that kind of action should happen by the caller of this private method, not the method itself...

`this.setActivateAutoFillOnPageLoadFromPolicy(true)` will actually set the value off? Hmm...

Reply:
naming handled in bae53cc5a8bff38ece17338d44be22f6155fa46d

Reply:
> it feels like that kind of action should happen by the caller of this private method, not the method itself...

Good catch; I completely agree; will resolve...

Reply:
Okay, I'm not sure where the inversion in the private method came from (it wasn't in the original state service method) - I think it was a copy-paste typo.

This should be in line with the original logic now:
66ce97cdbc57759748d3a964dab5463dbe50c22a